lib/DDG/Spice/Npm.pm

```diff
@@ -1,13 +1,17 @@
 package DDG::Spice::Npm;
-# ABSTRACT: Returns package information from npm package manager's registry.
+# ABSTRACT: Search npm for package information
 
 use strict;
 use DDG::Spice;
+use warnings;
 
-triggers startend => 'npm', 'nodejs';
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d';
+
+triggers startend => 'npm', 'nodejs', 'node package';
```

Comment:
@adampolar please add `node js` -- I see that appearing a lot in the autocomplete suggestions!